src/standalone/Vegetation/optimality_farquhar.jl

```diff
@@ -71,46 +71,35 @@ Base.eltype(::OptimalityFarquharParameters{FT}) where {FT} = FT
     )
 A constructor supplying default values for the FarquharParameters struct.
 """
-function OptimalityFarquharParameters{FT}(;
-    oi = FT(0.209),
-    ϕ = FT(0.6),
-    θj = FT(0.9),
-    f = FT(0.015),
-    sc = FT(5e-6),
-    pc = FT(-2e6),
-    Γstar25 = FT(4.275e-5),
-    Kc25 = FT(4.049e-4),
-    Ko25 = FT(0.2874),
-    To = FT(298.15),
-    ΔHkc = FT(79430),
-    ΔHko = FT(36380),
-    ΔHVcmax = FT(58520),
-    ΔHΓstar = FT(37830),
-    ΔHJmax = FT(43540),
-    ΔHRd = FT(46390),
-    c = FT(0.05336251),
-) where {FT}
-    mechanism = C3()
-    return OptimalityFarquharParameters{FT}(
-        mechanism,
-        Γstar25,
-        Kc25,
-        Ko25,
-        ΔHkc,
-        ΔHko,
-        ΔHVcmax,
-        ΔHΓstar,
-        ΔHJmax,
-        ΔHRd,
-        To,
-        oi,
-        ϕ,
-        θj,
-        f,
-        sc,
-        pc,
-        c,
+OptimalityFarquharParameters(::Type{FT}; kwargs...) =
+    OptimalityFarquharParameters(CP.create_toml_dict(FT); kwargs...)
+
+function OptimalityFarquharModel(toml_dict; kwargs...)
+    name_map = 
+    (;
+    :Jmax_activation_energy             => :ΔHJmax,
+    :intercellular_CO2_concentration    => :oi,
+    :CO2_compensation_point_25c         => :Γstar25,
+    :Farquhar_curvature_parameter       => :θj,
+    :kelvin_25C                         => :To,
+    :photosystem_II_quantum_yield       => :ϕ,
+    :O2_michaelis_menten                => :Ko25,
+    :CO2_michaelis_menten               => :Kc25,
+    :maximum_dry_soil_layer_depth       => :f,
```

Comment:
incorrect name here, will look up correct one

Reply:
In the description of `FarquharParameters`: f is the constant factor appearing the dark respiration term, equal to 0.015.
Looks like the `maximum_dry_soil_layer_depth` has the same default value. I will add it in as a separate parameter.